src/proxy/PartitionSpecificProxy.ts

```diff
@@ -43,7 +44,8 @@ export class PartitionSpecificProxy extends BaseProxy {
         serializationService: SerializationService,
         connectionManager: ClientConnectionManager,
         listenerService: ListenerService,
-        clusterService: ClusterService
+        clusterService: ClusterService,
+        connectionRegistry: ConnectionRegistry
```

Comment:
Do we really need `connectionManager` field here?

Reply:
No not really :)